jql-core/src/main/java/io/github/benas/jql/core/AnnotationMemberIndexer.java

```diff
@@ -38,9 +38,8 @@ public AnnotationMemberIndexer(MethodDao methodDao) {
     }
 
     public void index(AnnotationMemberDeclaration annotationMemberDeclaration, int typeId) {
-        int annotationMemberModifiers = annotationMemberDeclaration.getModifiers();
-        methodDao.save(new Method(annotationMemberDeclaration.getName(),
-                isPublic(annotationMemberModifiers), isStatic(annotationMemberModifiers), isFinal(annotationMemberModifiers), isAbstract(annotationMemberModifiers), false, typeId));
+        methodDao.save(new Method(annotationMemberDeclaration.getNameAsString(),
+                annotationMemberDeclaration.isPublic(), false, false, annotationMemberDeclaration.isAbstract(), false, typeId));
```

Comment:
annotation interface members can't be static or final.

Reply:
indeed